examples/boltzmann_wealth_model/requirements.txt

```
@@ -1 +1 @@
-mesa~=2.0
+mesa[viz]~=3.0.0b0
```

We will keep updating this one
```suggestion
mesa[viz]>=3.0.0b0
```